src/main/scala/org/constellation/serialization/ConstellationKryoRegistrar.scala

```diff
@@ -23,8 +23,9 @@ object ConstellationKryoRegistrar
         (classOf[EigenTrustAgents], 184),
         (classOf[StoredRewards], 185),
         (classOf[BuildInfoJson], 187),
-        (classOf[Signed[SnapshotProposal]], 1032),
-        (classOf[GossipMessage[Signed[SnapshotProposal]]], 1033),
+        (classOf[GossipMessage[Signed[SnapshotProposal]]], 1032),
+        (classOf[Signed[SnapshotProposal]], 1033),
+        (classOf[SnapshotProposal], 1033),
```

Comment:
Repeated id 1033 and flipped 1032 to 1033. Is that intentional?

Reply:
Non intentional, fixing it.